cypress/integration/shared.js

```diff
@@ -158,6 +158,9 @@ export const downloadFileWithChrome = (downloadButton, resultantFilename) => {
   cy.exec(`[ -f ${resultantFilename} ] && rm -rf ${resultantFilename}`, {
     failOnNonZeroExit: false,
   });
+  cy.wait(2 * 60 * 1000);
```

Comment:
Can we add a condition instead of the delay? If not, please write add a comment describing what do we wait for and preferably verify after this `wait()`.

Reply:
Currently the wait is a ruff estimate of what is happening in the backend. Nothing in UI to base the wait on. Still this is moved to the caller parent file.